src/varnode.jl

```diff
@@ -32,3 +32,32 @@ Checks if an [`AbstractRuleNode`](@ref) tree contains a [`VarNode`](@ref) with t
 """
 contains_varnode(rn::AbstractRuleNode, name::Symbol) = any(contains_varnode(c, name) for c ∈ rn.children)
 contains_varnode(vn::VarNode, name::Symbol) = vn.name == name
+
+"""
+	HerbCore.update_rule_indices!(c::ContainVarNodesSubtree, n_rules::Integer)
+
+This function serves as interface for `HerbCore.update_rule_indices!` on node type `VarNode`. Since `VarNode` doesn't change,
+this function does not perform any operations. 
+
+"""
```

Comment:
```suggestion
"""
     HerbCore.update_rule_indices!(c::ContainVarNodesSubtree, n_rules::Integer)

Update the rule indices of a `VarNode`. As `VarNode`s contain no indices, this function does nothing.
"""
```

While it's true this implements the interface, I don't think it's super useful to state in the docstring since it's already in the signature.